docs/config/index.md

```diff
@@ -366,21 +393,21 @@ Listen to port and serve API. When set to true, the default port is 55555
 - **Type:** `boolean`
 - **Default:** `false`
 
-Will call `.mockClear()` on all spies before each test
+Will call [`.mockClear()`](/api/#mockclear) on all spies before each test. This will clear mocks history, but not reset its implementation to default one.
```

Comment:
```suggestion
Will call [`.mockClear()`](/api/#mockclear) on all spies before each test. This will clear mock history, but not reset its implementation to the default one.
```